Makefile

```diff
@@ -92,14 +92,9 @@ build: ## Generate a binary in ./bin
 
 .PHONY: e2e-test-atlas
 e2e-test-atlas: build ## Run Atlas E2E tests
-	@echo "==> Running E2E tests..."
+	@echo "==> Running Atlas E2E tests..."
 	# the target assumes the MCLI-* environment variables are exported
 	go test -v -p 1 -parallel 1 -tags=e2e ./e2e/atlas...
-
-.PHONY: e2e-test-iam
```

Comment:
got it, the diff here made it look like this was gone but now I see we run two `go test command` this is, not ideal but let's do it for now